lib/mixpanel-node.d.ts

```diff
@@ -0,0 +1,159 @@
+declare const mixpanel: mixpanel.Mixpanel;
+
+declare namespace mixpanel {
+  export type DistinctId = string;
+
+  export type ErrorCallback = (err: Error | undefined) => any;
+  export type ErrorsCallback = (errors: [Error] | undefined) => any;
+
+  type Scalar = string | number | boolean;
+
+  export interface PropertyMap {
+    [key: string]: any;
+  }
+
+  export interface NumberMap {
+    [key: string]: number;
+  }
+
+  export interface Event {
+    event: string;
+    properties: PropertyMap;
+  }
+  export interface Modifiers {
+    $ip?: string;
+    $ignore_time?: boolean;
+    $time?: string;
+    $ignore_alias?: boolean;
+  }
+
+  export interface BatchOptions {
+    max_concurrent_requests?: number;
+    max_batch_size?: number;
+  }
+
+  export interface UnionData {
+    [key: string]: Scalar | Scalar[];
+  }
+
+  interface Mixpanel {
+    init(mixpanelToken: string, config?: PropertyMap): Mixpanel;
+
+    track(eventName: string, callback?: ErrorCallback): undefined;
+    track(eventName: string, properties: Object, callback?: ErrorCallback): undefined;
+
+    track_batch(events: Event[], options?: BatchOptions, callback?: ErrorsCallback): undefined;
+    track_batch(events: Event[], callback: ErrorsCallback): undefined;
+    track_batch(eventNames: string[], options?: BatchOptions, callback?: ErrorsCallback): undefined;
+    track_batch(eventNames: string[], callback?: ErrorsCallback): undefined;
+
+    import(eventName: string, time: Date | number, properties?: Object, callback?: ErrorCallback): undefined;
```

Comment:
or alternatively you can have a `PropertyBag` interface and use it instead of `Object`

Reply:
yep, this was actually supposed to use the ill-named PropertyMap. thanks for catching it!